backend/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.1'
```

Comment:
This file should in the root of the repo (or in `/dev/` subfolder of the root) because the point of this is to be able to easily launch the full project ; including the UI.